src/store/announcers/index.ts

```diff
@@ -0,0 +1,58 @@
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
+import { ResponseApi } from '@uspacy/sdk/lib/services/AnnouncersService/dto/announcers-dto';
+
+import { fetchAnnouncers } from './actions';
+import { IState } from './types';
+
+const initialState: IState = {
+	loadingAnnouncers: false,
+	errorLoading: false,
+	isBannerExists: false,
+	announcers: {
+		data: {
+			id: 0,
+			attributes: {
+				notifications: [],
+				widgets: [],
+				banner: null,
+			},
+		},
+		meta: {
+			pagination: {
+				page: 0,
+				pageSize: 0,
+				pageCount: 0,
+				total: 0,
+			},
+		},
+	},
+};
```

Comment:
не подобається ця структура зовсім, зайві поля і вкладеність, не потрібно зберігати таке як id: 0, attributes, та й announcers також можна викинути
зроби щось типу
```suggestion
const initialState: IState = {
	loadingAnnouncers: false,
	errorLoading: false,
	isBannerExists: false,
	notifications: [],
	widgets: [],
	banner: null,
	meta: {
		pagination: {
			page: 0,
			pageSize: 0,
			pageCount: 0,
			total: 0,
		},
	},
};
```

Reply:
+